### ISSUE TITLE
Create User/Role with keyboard only

### ISSUE THREAD

**Issue by sranka** [influxdata/chronograf#5952]
InfluxDB “Create User/Role” requires you to click the `Create` button, a user should be able to create the user with a keyboard only.

__Environment info:__

* Chronograf version: pre1.10
